src/modules/plus/filter_dynamicgpstext.yml

```diff
@@ -0,0 +1,321 @@
+      The # may be escaped with "\".
+    required: yes
+    readonly: no
+    default: >
+      Speed: #gps_speed_c#km/h\n
+      Distance: #gps_dist_c#m\n
+      Altitude: #gps_elev#m\n\n
+      GPS time: #gps_datetime_now#\n
+      GPS location: #gps_lat_c#, #gps_lon_c#
+    widget: text
+
+  - identifier: gps.file
+    title: File/URL
+    type: string
+    description: >
+      "The fullpath of file containing location (GPS) data. Supported extensions: .gpx, .tcx."
+    required: no
+    readonly: no
+    widget: fileopen
+    
+  - identifier: major_offset
+    title: Major offset
+    type: integer
+    description: >
+      An offset (in seconds) to be added to the video file to match it to the GPS track.
+      Most of the time this will at least need to be set to the timezone difference 
+      between the 2 files plus some seconds if the video recording device isn't 
+      precisely set to correct time. GPS time is always exact and in UTC. 
+      Use positive values if GPS is ahead of video and negative otherwise.
+    default: 0
+    readonly: no
+    required: no
+    mutable: yes
+    widget: text
+    
+  - identifier: minor_offset
+    title: Minor offset
+    type: integer
+    description: >
+      An extra offset (in seconds) to be also added to video file to match it to the GPS
+      track. The intended use of this value is to have a realtime fine control over the 
+      sync on a per clip basis (this way the major offset can be kept the same over 
+      multiple videos). 
+    default: 0
+    readonly: no
+    required: no
+    mutable: yes
+    widget: spinner
+    
+  - identifier: smoothing_value
+    title: Smoothing level
+    type: integer
+    description: >
+      How many GPS points to smooth in order to eliminate GPS errors.
+      A value of 0 only exposes the raw values from file.
+      A value of 1 does not smooth locations, it only calculates the extra 
+      fields (speed, distance, etc) it also interpolates missing values for
+      heart rate and altitude. 
+      After setting this value to non zero the interpolated hr/altitude
+      will remain in memory, reload the file using a value of 0 to get
+      back the pure raw data. 
+    default: 5
+    minimum: 0
+    readonly: no
+    required: no
+    mutable: yes
+    widget: text
+    
+  - identifier: gps_processing_start_time
+    title: GPS processing start time
+    type: string
+    description: >
+      A UTC date and time (formatted as "yyyy-MM-dd hh:mm:ss") from which to start
+      processing the gps file. Use if the GPS file has extra points before the video
+      and you want to ignore them (can be used to track the distance since begining 
+      of a segment).
+      If the string provided doesn't perfectly match the format it will be ignored.
+    default: "yyyy-MM-dd hh:mm:ss"
+    readonly: no
+    required: no
+    mutable: yes
+    widget: text
+    
+  - identifier: video_timelapse_speed
```

Comment:
Name suggestion: "speed_multiplier"

Also, the description seems unnecessarily complicated. It it just as simple that it multiplies the speed of the GPS time? Or is there something more tricky than that about it?

Reply:
Hmm, true. I was trying to emphasize that the multiplication is done on the seconds relative to file start and then I got sidetracked by one scenario in which the user would think that if he trimmed the clip in shotcut it would change the "start". But indeed, this is too much detail, it works as expected 99% of cases.